### ISSUE TITLE
Reduce duplicate ids in rich text (disallow ids?)

### ISSUE THREAD

**Issue by kevwalsh** [department-of-veterans-affairs/va.gov-cms#5421]
## User Story or Problem Statement

Allowing editors to specify ids in rich text, usually for headers, which lead to issues like duplicate ids and accessibility errors. [See this slack thread](https://dsva.slack.com/archives/CDHBKAL9W/p1621605179016900) as an example. 

Background: this was allowed originally because the FE had no means of assigning ids during the build. Once the FE added that, existing ids were left, and the functionality lives on. 

How might we reduce the likelihood of bad ids (like duplicates) going out in the editor? 

Do editors still need this? Most rich text fields with ids are in the Benefits Hub product. 

## Acceptance Criteria
- [ ] TBD

## Implementation steps
- [ ] Audit existing rich text fields for ids? 

## Design principles

Which [CMS design principle](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/cms/cms-product-design-principles.md) is at play?
- [ ] Purpose-driven: Creates an opportunity to involve the editor community in VA’s mission and content strategy goals.
- [ ] Efficient: Removes distractions and create clear, straightforward paths to get the job done.
- [ ] Approachable: Offers friendly guidance over authoritative instruction.
- [ ] Consistent: Reduce user’s mental load by allowing them to fall back on pattern recognition to complete tasks.
- [ ] Empowering: Provide clear information to help editors make decisions about their work.

### CMS Team

Please leave only the team that will do this work selected. If you're not sure, it's fine to leave both selected.

- [x] `Core Application Team`
- [x] `Product Support Team`

**Comment by EWashb** [comment]
@jilladams are benefit hubs a PW product?

**Comment by jilladams** [comment]
They are! Adding our label.

**Comment by laflannery** [comment]
@jilladams and @EWashb duplicate IDs are an issue throughout the CMS, not just on PW products.  Can I ask why this was assigned to the PW team and not the CMS team?

**Comment by jilladams** [comment]
I think bc the specific ask in comments above re: Benefit hubs.

**Comment by laflannery** [comment]
I'm going to close this in favor of a more recent/general ticket I've created (#13204).  If that seems incorrect we can reopen this but this is an issue in every WSYWIWYG not just Benefits
